doc/project/js-environments.md

```diff
@@ -90,7 +90,7 @@ jsEnv := PhantomJSEnv(args = Seq("arg1", "arg2")).value
 {% endhighlight %}
 
 For more options of the PhantomJS environment, see
-[the Scaladoc of `PhantomJSEnv`]({{ site.production_url }}/api/sbt-scalajs-env-phantomjs/1.0.0/org/scalajs/jsenv/phantomjs/sbtplugin/PhantomJSEnvPlugin$$autoImport$.html).
+[the Scaladoc of `PhantomJSEnv`]({{ site.production_url }}/api/sbt-scalajs-env-phantomjs/{{ site.versions.scalaJS }}/org/scalajs/jsenv/phantomjs/sbtplugin/PhantomJSEnvPlugin$$autoImport$.html).
```

Comment:
This should remain hard-coded 1.0.0. The version of sbt-scalajs-env-phantomjs is independent of the version of Scala.js.

Reply:
All right, thought it was a typo! Fixed in 1349916893f6b79555756ee40b5fc33ff9b99d44